relational_operators/DropTableOperator.cpp

```diff
@@ -32,9 +34,21 @@ namespace quickstep {
 bool DropTableOperator::getAllWorkOrders(WorkOrdersContainer *container) {
   if (blocking_dependencies_met_ && !work_generated_) {
     work_generated_ = true;
+
+    std::vector<block_id> relation_blocks(relation_.getBlocksSnapshot());
+
+    // DropTableWorkOrder only drops blocks, if any.
     container->addNormalWorkOrder(
-        new DropTableWorkOrder(relation_.getID(), only_drop_blocks_),
+        new DropTableWorkOrder(std::move(relation_blocks)),
         op_index_);
+
+    // TODO(zuyu): move the following code to a better place.
```

Comment:
These code changes `Catalog`, and seems not related to `getAllWorkOrders`.

Similar to `CreateTableOperator::getAllWorkOrders`, which executes some tasks, instead of generating work orders.